coalib/bearlib/languages/definitions/PyPyLog.py

```diff
@@ -0,0 +1,6 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class PyPyLog:
```

Comment:
Need to see if this is a useful format for coala . Same with py tracebacks that follow.